### PR TITLE
fix: character count should includes spoiler text

### PR DESCRIPTION
**Pull request by Mini-ghost**
resolves: #1512

In Mastodon, content warning (spoiler text) will be included in the character count

mastodon

But not currently
main

After this PR, when the content warning (spoiler text) is activated, it will be included in the word count, and it will be ignored when it is closed.

https://user-images.githubusercontent.com/39984251/215755701-69f024c9-969f-403f-8fc6-90f535b40079.mp4

### CODE REVIEW

File: composables/masto/publish.ts [hunk 1]
```diff
@@ -21,6 +21,17 @@ export function usePublish(options: {
   const isExpanded = $ref(false)
   const failedMessages = $ref<string[]>([])
 
+  const publishSpoilerText = $computed({
+    get() {
+      return draft.params.sensitive ? draft.params.spoilerText : ''
+    },
+    set(val) {
+      if (!draft.params.sensitive)
+        return
```
Comment (danielroe) [inline composables/masto/publish.ts:30]: Do we need this?

```suggestion
```

Comment (Mini-ghost) [reply composables/masto/publish.ts:30]: No, just to make the logic more rigorous, is it recommended to delete it?

Comment (danielroe) [reply composables/masto/publish.ts:30]: I can't really think when this would be called, so I think it probably doesn't matter either way.

**Review by danielroe — APPROVED** [review]

File: composables/masto/publish.ts (hunk 1 above)
Comment (Mini-ghost) [reply composables/masto/publish.ts:30]: @danielroe  Got it, Normal operation here would never call. I wish I could remove this if you prefer cleaner code.

**Pull request merged**